dao/src/main/java/se/fortnox/reactivewizard/db/statement/SelectStatementFactory.java

```diff
@@ -25,7 +28,19 @@ protected void executeStatement(Connection connection, Object[] args, FluxSink f
             try (ResultSet resultSet = statement.executeQuery()) {
                 while (resultSet.next()) {
                     if (fluxSink != null) {
-                        fluxSink.next(deserializer.deserialize(resultSet));
+                        var value = deserializer.deserialize(resultSet);
+                        if (value == null) {
+                            throw new NullPointerException("""
```

Comment:
Is it possible to add the query and/or typeinfo of the method to the error, otherwise you have no clue to what query is causing this?

Reply:
good point, I added the parameterized query.